interoperator/controllers/multiclusterdeploy/provisioner/provisioner_controller.go

```diff
@@ -547,6 +570,8 @@ func (r *ReconcileProvisioner) SetupWithManager(mgr ctrl.Manager) error {
 	interoperatorCfg := cfgManager.GetConfig()
 	r.cfgManager = cfgManager
 
+	metrics.Registry.MustRegister(clusterMetric)
```

Comment:
Hi @anoopjb , or the clusterMetric gets uploaded to Prometheus here? If so, then the clusterMetric comment i added earlier is invalid (clusterMetric is still an in-mem variable)?